sympy/physics/optics/polarization.py

```diff
@@ -86,50 +86,41 @@
 
 
 def jones_vector(psi, chi):
-    u"""A Jones vector corresponding to a polarization ellipse with `psi` tilt,
-    and `chi` circularity.
-
-    Parameters
-    ----------
-    ``psi`` : numeric type or sympy Symbol
-        The tilt of the polarization relative to the `x` axis.
-
-    ``chi`` : numeric type or sympy Symbol
-        The angle adjacent to the mayor axis of the polarization ellipse.
-
-
-    Returns
-    -------
-    Matrix
-        A Jones vector.
+    u"""
```

Comment:
xref https://github.com/sympy/sympy/pull/17715#discussion_r334064933 and https://github.com/numpy/numpydoc/issues/241

[PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) is quite clear that, for multi-line docstrings:
> "The summary line may be on the same line as the opening quotes or on the next line."

Unfortunately, the [numpydoc Docstring Guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections) does not clarify that this is an option for multi-line docstrings and instead gives some confusing examples of one-line docstrings (with and without signatures). It is not until later in the guide that there are some examples of multi-line docstrings, and it is left to the reader to draw the inference.

Many large projects prefer to have the summary line on the line after the opening quotes. Some reasons I've heard:
- It improves readability
- It gives greater visibility to whether the docstring is a unicode or raw string
- It gives the summary line 3-4 extra characters when attempting to fit on a single line